support-frontend/assets/pages/[countryGroupId]/events/events.tsx

```diff
@@ -31,10 +31,20 @@ const columns = css`
 `;
 
 const shorterBoxMargin = css`
-	border-radius: ${space[2]}px;
+	border-radius: ${space[4]}px;
 	:not(:last-child) {
 		margin-bottom: ${space[4]}px;
 	}
+
+	> div {
+		padding: 0;
+	}
+
+	// ticket tailor widget (class:box-color-background) margin
+	// needs removal, its container we can reduce to hide
+	> div div div {
+		margin: -2px -2px -8px -2px;
+	}
```

Comment:
This is quite a confusing solution - I am assuming it's because we're using Source, and we need to target the `div`s that creates?

Could we explore not using source?

Either way - we might want to change the name `shorterBoxMargin` to something more descriptive.

Reply:
Have renamed the css to `narrowBoxMarginAndPadding` which is a bit more descriptive.

Struggled to access css on div's within the ticket tailor widget itself (presume does not exist until runtime?) thus could not access the class:`box-color-background` directly to override its css.

Reply:
Will put in as-is and revisit if a better way appears.